packages/synapse-react-client/src/lib/utils/hooks/useOverlay.tsx

```diff
@@ -70,35 +77,33 @@ export function useOverlay(
 
   const OverlayComponent = useCallback(
     () => (
-      <Overlay
-        target={targetRef.current}
-        show={isShowing}
-        placement={placement}
+      <Popover
```

Comment:
Use the MUI version. The bootstrap version didn't work in MUI modals

Reply:
Nice! Removes another bootstrap-4-backport instance too